core/domain/feedback_jobs_one_off.py

```diff
@@ -100,3 +102,88 @@ def _cache_last_nonempty_message_author_id(
             thread_model.last_nonempty_message_author_id = message_author_id
             return True
         return False
+
+
+class CleanUpFeedbackAnalyticsModelModelOneOffJob(
```

Comment:
This is not still being done in codebase and instead of running clean ups at regular intervals - this should be fixed via #10809

Reply:
Fixed it instead, no need for a TODO! :)

Also, added comments regarding deletion here and other PRs.

Opened a separate one for PRs already merged: #10827.

Reply:
Woohoo, thanks @ankita240796 -- that's great!

Reply:
Do you still need the "Do not delete this job" comment, btw?

Also does this apply to the CleanUpGeneralFeedbackThreadModelOneOffJob, too?